src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,21 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+      <UserInfo user={post.user} />
```

Comment:
We can get null instead of user so lets use conditional rendering here

Reply:
done